pittAPI.py

```diff
@@ -411,10 +412,9 @@ def get_person(self, query, maxPeople=10):
                 personurl = str(''.join(url))
 
                 if personurl.lower().startswith("https://") and hasattr(ssl, '_create_unverified_context'):
-                    ct = ssl._create_unverified_context()
-                    f = urlopen(personurl, context=ct)
+                    f = s.get(personurl, verify=False)
                 else:
-                    f = urlopen(personurl)
+                    f = s.get(personurl)
 
                 html = f.read()
```

Comment:
We now have an error here, `AttributeError: 'Response' object has no attribute 'read'`. Please confirm on how to get the data for a Response object.